dashboard/app/views/projects/index.html.haml

```diff
@@ -2,7 +2,6 @@
 
 = render partial: 'shared/angular'
 
-%script{src: "/shared/js/angularProjects.js"}
+%script{src: asset_path('angularProjects.js') }
```

Comment:
Not currently (and this is perhaps confusing). AngularProjects.js does not currently go through a JS pipeline, and we're actually including the file from cdo/shared/js.

By contrast shared.js is a file created via gulp that ends up in cdo/dashboard/public/shared/js, and includes both a minified and unminified version.